camelot/view/controls/calculator.py

```diff
@@ -28,7 +28,7 @@
 #  ============================================================================
```

Comment:
ik vroeg me al af waar die backspace vandaan kwam, hier dus.

mischien smijten we beter gewoon die calculator er uit ...

Reply:
tbd

Reply:
Ik heb het probleem gevonden. Er staat een versie van dit font geinstalleerd op CentOS: /usr/share/fonts/fontawesome/fontawesome-webfont.ttf

Deze versie is bevat niet alle icoontjes (oudere versie?) en heeft "FontAwesome" als font family name.

Het ttf file dat we laden bevat wel de nieuwe icoontjes, maar heeft font family name "Font Awesome 5 Free". Door deze font family name te gebruiken, is het probleem opgelost.

Het vfinance/art/qml/editor.qml bestand bevat ook de verkeerde font family name, ik zal deze ook aanpassen. Dit kan ook de problemen verklaren onder windows (cfr email).

Reply:
just ja, in vf/art/fonts/README staat een mogelijke oplossing voor een gelijkaardig probleem.

het komt er op neer dat je de font naam aanpast, zodat het niet in conflict is met mogelijk reeds geinstalleerde fonts

Reply:
dat laatste was puur fyi heh

Reply:
Ja, dat is idd een gelijkaardig probleem. Dit kan nog van pas komen als we de regular en solid versie door elkaar willen gebruiken vermits beide de naam "Font Awesome 5 Free" hebben.

Met aangepaste editor.qml, werken de icoontjes van de notification editor ook onder windows:

before:
![fontawesome-before](https://user-images.githubusercontent.com/65219298/93183049-cb591c00-f73a-11ea-8adc-cf88b4c0ba26.png)
after:
![fontawesome-after](https://user-images.githubusercontent.com/65219298/93183066-cf853980-f73a-11ea-9d19-cf310ce611c0.png)